frontend/src/views/application_form/actions.js

```diff
@@ -1,6 +1,6 @@
 import { UPDATE_FIELD } from "./constants";
 import { error, success } from "react-notification-system-redux";
-import { errorProps } from "../notifications/constants";
+import { errorProps } from "../../modules/notifications/constants";
 import { apiPOST } from "../../libs/apiUtils";
 
 // an action generator function that returns an action object
```

Comment:
actions defined in a view should only relate to UI state, not model state. All actions for changing/updating model state and interacting with the API are in `/api`. There is no reason a view should ever import `apiPOST`, for example.

Reply:
I refactored part of the actions into api/actions.  Let me know of what you think.